### PR TITLE
Autostart server and admin console + logs to stdout

### PR DESCRIPTION
**Pull request by berezins**
- Updated to CentOS 7
- Option to disable access log (to redirect it to /dev/null instead of stdout)
- Updated README to reflect all changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,12 +12,54 @@ to build the docker container image
...
+
+### Manually (just for development/experiments/tests)
+
 Run and start bash (we will be running the server from here). Note: We have to use --service-ports because by default, the [`docker-compose run` command does not map ports](https://docs.docker.com/compose/reference/run/).
 ```
 docker-compose run --service-ports ams bash
 ```
 
-to run the server:
+to run the server (with logging to stdout):
```
Comment (ultrasaurus) [inline README.md]: the following commands don't log to stdout in my experience -- did you mean to change the lines below?

File: start.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    done
+}
+
+
+for log in $logs; do
+    fifo=./logs/$log.00.log
+    dir=`dirname $fifo`
+    mkdir -p $dir
+    chmod 0777 $dir
+    rm -f $fifo # just make sure regular file does not exists
```
Comment (ultrasaurus) [inline start.sh]: nit: exists => exist

File: README.md (hunk 1 above)
Comment (berezins) [reply README.md]: Mistake, fixed, force-pushed

File: start.sh (hunk 2 above)
Comment (berezins) [reply start.sh]: fixed, force-pushed

**Pull request merged**